components/Listen/Listen.tsx

```diff
@@ -287,7 +292,7 @@ const Listen = ({ config, data }: IListenPageProps) => {
               )}
             </div>
             <div className={styles.podcastEpisodes}>
-              <EpisodeList onEpisodeClick={handleEpisodeClick} />
+              {view.indexOf('podcast') !== -1 && renderPlaylist}
```

Comment:
Only render the playlist when we are on the podcast view.  Makes sense!

Reply:
I am looking into how to render the playlist so it is just as performant. Don't merge this yet. I may have a better solution.

Reply:
Your find of the Head code being hammered help me find the performance issue. The ThemeVars component, which is used by a lot of components to establish css variables in the `:root` was using the Head component to add the style elements to the DOM. I removed the use of Head component in ThemeVars and performance is mush better, even in playlist view.